syntax="proto3";
package SnowK;

// 生成通用的服务接口
// 用途: 在不同的实现中重用相同的服务接口定义，并且可以更容易地进行单元测试
option cc_generic_services = true;

message EchoRequest
{
    string message = 1;
}

message EchoResponse
{
    string message = 1;
}

service EchoService
{
    rpc Echo(EchoRequest) returns (EchoResponse);
}